render/helpers.go

```diff
@@ -86,6 +86,55 @@ func DateFormatYYYYMMDD(s string) template.HTML {
 	return template.HTML(t.Format("2006/01/02"))
 }
 
+// DatePeriodFormat will format a time-series date period string to a human accessible format e.g.
+// "2019 JAN-FEB" to "Jan - Feb 2019"
+// "2010 Q1" to "Jan - Mar 2010"
+func DatePeriodFormat(s string) string {
```

Comment:
Just checking but do we match Quarters based on calendar year or fiscal year? E.g. Q1 = Apr - Jun etc instead

Reply:
As per convo, throwing findings after checking into this comment for any future reference:

https://www.ons.gov.uk/economy/grossdomesticproductgdp/bulletins/gdpmonthlyestimateuk/february2020

> Q1 refers to Quarter 1 (Jan to Mar), Q2 refers to Quarter 2 (Apr to June), Q3 refers to Quarter 3 (July to Sept) and Q4 refers to Quarter 4 (Oct to Dec).